packages/framework/tests/Unit/Component/Router/Security/RouteCsrfProtectorTest.php

```diff
@@ -18,35 +19,31 @@ public function testSubRequest(): void
         $annotationReader = new AnnotationReader();
         $tokenManagerMock = $this->createMock(CsrfTokenManager::class);
 
-        $eventMock = $this->getMockBuilder(ControllerEvent::class)
-            ->disableOriginalConstructor()
-            ->setMethods(['isMasterRequest', 'getController'])
-            ->getMock();
-        $eventMock->expects($this->atLeastOnce())->method('isMasterRequest')->willReturn(false);
-        $eventMock->expects($this->never())->method('getController');
+        $event = new ControllerEvent(
+            $this->createMock(HttpKernelInterface::class),
+            new ControllerProtected(),
+            new Request(),
+            HttpKernelInterface::SUB_REQUEST,
+        );
 
         $routeCsrfProtector = new RouteCsrfProtector($annotationReader, $tokenManagerMock);
-        $routeCsrfProtector->onKernelController($eventMock);
+        $routeCsrfProtector->onKernelController($event);
     }
 
     public function testRequestWithoutProtection(): void
     {
         $annotationReader = new AnnotationReader();
         $tokenManagerMock = $this->createMock(CsrfTokenManager::class);
 
-        $eventMock = $this->getMockBuilder(ControllerEvent::class)
-            ->disableOriginalConstructor()
-            ->setMethods(['isMasterRequest', 'getController', 'getRequest'])
-            ->getMock();
-        $eventMock->expects($this->atLeastOnce())->method('isMasterRequest')->willReturn(true);
-        $eventMock
-            ->expects($this->atLeastOnce())
-            ->method('getController')
-            ->willReturn([DummyController::class, 'withoutProtectionAction']);
-        $eventMock->expects($this->never())->method('getRequest');
+        $event = new ControllerEvent(
+            $this->createMock(HttpKernelInterface::class),
+            new ControllerNotProtected(),
+            new Request(),
+            HttpKernelInterface::MASTER_REQUEST,
+        );
 
         $routeCsrfProtector = new RouteCsrfProtector($annotationReader, $tokenManagerMock);
-        $routeCsrfProtector->onKernelController($eventMock);
+        $routeCsrfProtector->onKernelController($event);
```

Comment:
this is the same story as `testSubRequest`, so could you please assert true here as well? :slightly_smiling_face:

Reply:
Fixed here: https://github.com/shopsys/shopsys/pull/2490/commits/2e480f6a69cef675227ae735ee86e90ae4461a27